pkg/env/process_pruning.go

```diff
@@ -1,8 +1,35 @@
 package env
 
+import "time"
+
 var (
 	// ProcessPruningEnabled toggles whether process pruning should be done periodically using heuristics
 	// This may be useful in certain large environments, but is fairly expensive as it requires a full
 	// sweep over the database
 	ProcessPruningEnabled = RegisterBooleanSetting("ROX_PROCESS_PRUNING", false)
+
```

Comment:
I think we need a slight refactor here.  This file is `process_pruning` which relates to a specific type of pruning based on heuristics in process indicators.  These new flags can relate to any pruning.  I think the new flags should either be moved to `postgres_default_timeout.go` where we already have some pruning vars or create a new `pruning.go`.

Reply:
Makes sense, moved it into the `postgres_default_timeout`.